### DISCUSSION TITLE
Discord link is expired again.

### DISCUSSION THREAD

**Discussion by rahat2134** [com-lihaoyi/mill#3727] [General]
@lefou , link is expired again -  <img width="1118" alt="Screenshot 2024-10-13 at 10 35 11 AM" src="https://github.com/user-attachments/assets/8c7a6f90-8b51-41c5-acdb-5ba4839036ee">  PTAL.

/cc @lihaoyi 

I checked a discussion was opened previously but it was closed, Hence I started a new one.
Thanks.

**Comment by lefou** [comment]
I can't do anything about it, really. I already checked the "never expire" option when creating the invite-link. Honestly, I think discord isn't the right platform for Mill community. It's the status quo, but I'm not going the upload an invite link every two weeks. I have better things to do with my time.

**Comment by rahat2134** [comment]
@lefou, can we try Google Chat or Slack?

Please prompt me if I can help, I know you would have some important commitments to fulfil.

Thanks :)

or is there a way someone else can update this link? May be any other member?

**Comment by lefou** [comment]
https://discord.com/channels/632150470000902164/632628729029328947/1295336376551608321

**Comment by rahat2134** [comment]
@lefou, I don't know why these invite links are some different. Generally invite links look like this - 
<img width="204" alt="Screenshot 2024-10-14 at 5 07 23 PM" src="https://github.com/user-attachments/assets/18d5c740-5950-43f1-a8d6-9006169bf2e5">

When I am using this invite link, It says - Invalid :(
<img width="427" alt="Screenshot 2024-10-14 at 5 08 04 PM" src="https://github.com/user-attachments/assets/8f19eb28-7cd6-4552-940d-49379264394e">
